cypress/plugins/index.js

```diff
@@ -1,13 +1,110 @@
 const getChallenge = require('../../curriculum/getChallenges');
+const env = require('../../config/env.json');
 
 module.exports = on => {
   on('task', {
     // return an array of all challenge paths
     //  superBlock, block,  this are unused variables in the condition Husky doesn't like that
-    getAllChallengePaths({ lang }) {
-      const curriculum = getChallenge.getChallengesForLang(lang);
+    getCurriculum({ lang }) {
+      let curriculum;
```

Comment:
This needs to be outside the function's scope or it will always be `undefined` when it's used in the if statement.